packages/cli/commands/feedback.js

```diff
@@ -0,0 +1,34 @@
+const open = require('open');
+
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const {
+  FEEDBACK_OPTIONS,
+  FEEDBACK_URLS,
+} = require('@hubspot/cli-lib/lib/constants');
+const { logger } = require('@hubspot/cli-lib/logger');
+
+const {
+  feedbackTypePrompt,
+  bugPrompt,
+  generalPrompt,
+} = require('../lib/prompts/feedbackPrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.feedback';
+
+exports.command = 'feedback';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async () => {
+  const { type } = await feedbackTypePrompt();
+
+  const promptForType =
```

Comment:
I think you can actually handle this dynamic "shouldOpen" prompt all within the initial `feedbackTypePrompt()` prompt. I linked the docs below, but check out `message`. It supports a function value where you can dynamically choose what to show.